FakeNFT/Modules/NTFDetails/NFTDetailsViewModel.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  NFTDetailsViewModel.swift
+//  FakeNFT
+//
+//  Created by Kirill on 25.06.2023.
+//
+
+protocol NFTDetailsViewModel {
+    var imageURL: String { get }
+    var sectionName: String { get }
+    var sectionAuthor: String { get }
+    var sectionDescription: String { get }
+    var nfts: Box<[NFT]> { get }
+    func selectedNft(index: Int)
+    func selectFavourite(index: Int)
+}
+
+final class NFTDetailsViewModelImpl: NFTDetailsViewModel {
+    private(set) var nfts: Box<[NFT]>
+    let imageURL: String
+    let sectionName: String
+    let sectionAuthor: String
+    let sectionDescription: String
+    private var profile: ProfileModel
+
+    private let nftStorageService: NFtStorageService
+    private let nftNetworkService: NFTNetworkService
+
+    init(nftStorageService: NFtStorageService, nftNetworkService: NFTNetworkService, details: NFTDetails) {
+        self.nftStorageService = nftStorageService
+        self.nftNetworkService = nftNetworkService
+        imageURL = details.imageURL
+        sectionName = details.sectionName
+        sectionAuthor = details.sectionAuthor
+        sectionDescription = details.sectionDescription
+        profile = details.profile
+
+        nfts = .init(details.items.map { NFT($0) })
+
+        nfts.value.forEach { nft in
+            if let index = details.profile.likes.firstIndex(where: { nft.id == $0}) {
```

Comment:
У фигурных скобок должно быть пространство: 
```suggestion
            if let index = details.profile.likes.firstIndex(where: { nft.id == $0 }) {
```
тут и далее в модуле часто такое нарушение стиля вижу